subsys/net/lib/download_client/src/http_download_client.c

```diff
@@ -49,71 +56,126 @@ static int socket_timeout_set(int fd)
 	return 0;
 }
 
-static int resolve_and_connect(const char *const host, const char *const port,
-			       u32_t family, u32_t proto)
+static int socket_sectag_set(int fd, int sec_tag)
+{
+	int err;
+	int verify;
+	sec_tag_t sec_tag_list[] = { sec_tag };
+
+	enum {
+		NONE = 0,
+		OPTIONAL = 1,
+		REQUIRED = 2,
+	};
+
+	verify = OPTIONAL;
```

Comment:
Why optional? I think that in a proper application we should always verify peer certificate.

Reply:
Resolved offline

Reply:
ok